insights/parsers/certificates_enddate.py

```diff
@@ -90,6 +104,6 @@ def expiration_date(self, path):
         if path_date:
             try:
                 path_datetime = datetime.strptime(path_date, '%b %d %H:%M:%S %Y')
-                return self.ExpirationDate(path_date, path_datetime)
+                return CertificatesInfo.ExpirationDate(path_date, path_datetime)
```

Comment:
@xiangce why change this parser if it is being deprecated?  Since it can't change the output and break backward compatibility, is this necessary?